app/policies/booking_policy.rb

```diff
@@ -7,4 +7,8 @@ def resolve
   def create?
     true
   end
+
+  def show?
```

Comment:
@K-HEG this needs to be checked, Bookings should only be shown to the users requesting it and the artist receiving it